libtelemetry/src/main/java/com/mapbox/android/telemetry/NavigationMetadata.java

```diff
@@ -35,27 +35,35 @@
   private Integer originalEstimatedDistance = null;
   private Integer originalEstimatedDuration = null;
   private String audioType;
-  private Integer stepCount = null;
+  private int stepCount;
   private Integer originalStepCount = null;
   private String device;
   private String locationEngine;
-  private Integer volumeLevel;
-  private Integer screenBrightness;
+  private int volumeLevel;
+  private int screenBrightness;
   private String applicationState;
   private Boolean batteryPluggedIn;
-  private Integer batteryLevel;
+  private int batteryLevel;
   private String connectivity;
+  private String tripIdentifier;
+  private int legIndex;
+  private int legCount;
+  private int stepIndex;
+  private Integer voiceIndex = null;
+  private Integer bannerIndex = null;
+  private int totalStepCount;
 
   public NavigationMetadata(Date startTimestamp, int distanceCompleted, int distanceRemaining, int durationRemaining,
-                            String sdKIdentifier, String sdkVersion, int eventVersion, String sessionIdentifier,
+                            String sdkIdentifier, String sdkVersion, int eventVersion, String sessionIdentifier,
                             double lat, double lng, String geometry, String profile, boolean isSimulation,
-                            String locationEngine, int absoluteDistanceToDestination) {
+                            String locationEngine, int absoluteDistanceToDestination, String tripIdentifier,
```

Comment:
This broke SemVer and should have been released as a major version (was released as patch `3.2.1`) or backporting the old logic and deprecating the old/to be removed constructor. We've noticed when Testing Maps SDK `6.6.0` releases downstream 👀 https://github.com/mapbox/mapbox-navigation-android/pull/1310 cc @andrlee @electrostat

Reply:
Good catch @Guardiola31337. At a first glance this seems to only affect the Nav SDK but cc'ing @tobrun here to confirm the Maps SDK side.